src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1546,10 +1546,30 @@ public void Copy()
         }
 
         /// <summary>
-        ///     Paste ISelectable objects from the clipboard to the workspace.
+        ///     Paste ISelectable objects from the clipboard to the workspace in top left corner
         /// </summary>
         public void Paste()
         {
+            // Move all of the notes and nodes such that they are aligned with
```

Comment:
just curious? is there a reason we are keeping this method around and not always creating the paste location near the mouse position?